<launch>
  <node pkg="serial_logger" name="serial_logger_node" type="serial_logger_node">
    <param name="port" value="/dev/ttyUSB0" />
    <param name="baudrate" type="int" value="115200" />
    <param name="charsize" type="int" value="8" />
    <!-- Send two stop bits, else one -->
    <param name="stopbits" type="bool" value="false" />
    <!-- Enable parity checking -->
    <param name="parityenable" type="bool" value="false" />
    <!-- Enable software flow control -->
    <param name="xonxoff" type="bool" value="false" />
    <!-- Enable RTS/CTS (hardware) flow control -->
    <param name="rtscts" type="bool" value="false" />
  </node>

</launch>
